config/desktop-development.json

```diff
@@ -97,7 +97,6 @@
 		"oauth": true,
 		"perfmon": false,
 		"persist-redux": true,
-		"plans/jetpack-backup": false,
 		"plans/jetpack-scan": false,
```

Comment:
We're not using this flag as well. It was added in an anticipation that we're going to be adding both Backup and Scan in the first iteration and it was not the case. Let's get rid of it here and in other config files.

Reply:
Good call! I'll remove in another PR 👍

Reply:
#38284